release.gradle

```diff
@@ -42,7 +54,7 @@ def verifySdkVersion(deployment) {
         while (entry != null) {
             if (entry.getName().equals("com/dropbox/core/sdk-version.txt")) {
                 def version = jar.text.trim();
-                if (!(version ==~ /\d+\.\d+\.\d+/)) {
```

Comment:
From my understanding, we still want the `\` in front of the `.` chars, to avoid regex matching something like `4a0b0c`

Reply:
You're right, good catch!